src/main/java/net/minestom/server/listener/BlockPlacementListener.java

```diff
@@ -92,10 +92,11 @@ public static void listener(ClientPlayerBlockPlacementPacket packet, Player play
         final Point placementPosition = blockPosition.add(offsetX, offsetY, offsetZ);
 
         if (!canPlaceBlock) {
-            // Send a block change with AIR as block to keep the client in sync,
+            // Send a block change with the real block in the instance to keep the client in sync,
             // using refreshChunk results in the client not being in sync
             // after rapid invalid block placements
-            player.getPlayerConnection().sendPacket(new BlockChangePacket(placementPosition, Block.AIR));
+            final Block block = interactedChunk.getBlock(placementPosition);
```

Comment:
I would suggest retrieving the block from the instance, as it comes with additional synchronization not present in the chunk method.

Reply:
Got it, just changed it.